src/permissions/components/Restricted.js

```diff
@@ -26,6 +26,7 @@ const Restricted = props => {
   const {
     permission,
     resource,
+    forUnallowedUsers = false,
```

Comment:
suggestion: disallowed instead of unallowed.

Reply:
i did `forUnallowedUsers -> toDisallowedUsers` since i think that's a bit more clear phrasing (restrict for maybe sounds like restricting unallowed users from seeing the thing, instead of restricting seeing the thing to them). y is phrasing so hard